dask_jobqueue/slurm.py

```diff
@@ -73,8 +73,8 @@ def __init__(self, queue=None, project=None, walltime=None, job_cpu=None, job_me
         # SLURM header build
         if self.name is not None:
             header_lines.append('#SBATCH -J %s' % self.name)
-            header_lines.append('#SBATCH -e %s.err' % self.name)
-            header_lines.append('#SBATCH -o %s.out' % self.name)
+            header_lines.append('#SBATCH -e %s-%%j.err' % self.name)
```

Comment:
Just curious, I have wondered before, why not keep the default name? Is the default name not something very similar?

Reply:
It was my first solution too, but I found that Slurm uses slurm-$JOB_ID.out as default. This is not too bad, but it misses the job name... So I'm hesitating here.